pkg/target/http.go

```diff
@@ -76,20 +78,21 @@ func getHeaders(headers string) (map[string]string, error) {
 	return parsed, nil
 }
 
-func addHeadersToRequest(request *http.Request, headers map[string]string) {
-	if headers == nil {
```

Comment:
Have we tested if this has an effect? 

I'm not sure why I added this check but I don't think I would have if not for some weird behaviour when none are provided.

Reply:
If headers are nil the for loop is not being executed. The same when headers are an empty map. I'll make sure there are test cases for it, but also happy to add the nil check back in.

Reply:
No as long as we have checked, I'm happy. I'm pretty sure I added it but I have no idea why :D